standup/status/tests/test_commands.py

```diff
@@ -35,7 +35,7 @@ def test_find_user(db):
             'Searching for: jimbob',
             '-----',
             'user_id  username',
-            '1        jimbob',
+            '%-5d    jimbob' % user.id,
```

Comment:
For some reason, the user id isn't always 1. I suspect it's because either I was kicking off the tests in a way that reused the db or something. Anyhow, this is a good fix to make the tests less fragile.